management/src/main/java/org/kaazing/gateway/management/config/ServiceDefaultsConfigurationBeanImpl.java

```diff
@@ -95,35 +99,120 @@ public String getAcceptOptions() {
                     jsonOptions.put("ssl.verify-client", "none");
                 }
 
-                jsonOptions.put("ws.maximum.message.size", acceptOptions.get("ws.maximum.message.size"));
+                jsonOptions.put("ws.maximum.message.size", acceptOptions.remove("ws.maxMessageSize"));
```

Comment:
They are explicitly ignored before processing the rest of the options.  See the loop towards the bottom of the method.